src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -36,6 +36,7 @@
 import logging
 import cart_logparse
 import cart_logtest
+import shlex
```

Comment:
(pylint-reimported) Reimport 'shlex' (imported line 39)
(pylint-ungrouped-imports) Imports from package shlex are not grouped